src/examples/USEnrichmentExample.cs

```diff
@@ -0,0 +1,87 @@
+namespace Examples
+{
+	using System;
+	using System.Net;
+	using System.Collections.Generic;
+	using System.IO;
+    using System.Linq;
+    using SmartyStreets;
+	using SmartyStreets.USEnrichmentApi;
+    using System.Reflection;
+
+	internal static class USEnrichmentExample
+	{
+		public static void Run()
+		{
+            // specifies the TLS protocoll to use - this is TLS 1.2
+            const SecurityProtocolType tlsProtocol1_2 = (SecurityProtocolType)3072;
```

Comment:
Typically constants are declared on the class level in C#. Here, it might be better to declare it not as a constant but as a "private static readonly"